internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -67,12 +72,26 @@ const (
 	BMH_AGENT_IGNITION_CONFIG_OVERRIDES = "bmac.agent-install.openshift.io/ignition-config-overrides"
 	MACHINE_ROLE                        = "machine.openshift.io/cluster-api-machine-role"
 	MACHINE_TYPE                        = "machine.openshift.io/cluster-api-machine-type"
+	MCS_CERT_NAME                       = "ca.crt"
 )
 
 var (
 	InfraEnvImageCooldownPeriod = 60 * time.Second
 )
 
+const certificateAuthoritiesIgnitionOverride = `{
+	"ignition": {
+	  "version": "3.1.0",
```

Comment:
Is this field mandatory? I'm wondering if there re could be a version mismatch between this one and some other ignition generated elsewhere.

Reply:
Yes, `version` field is mandatory. Ref https://coreos.github.io/ignition/configuration-v3_1/